src/main/webapp/js/finscholar/collegelistview/collegelistview.soy

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{namespace finscholar.collegelistview.templates}
+{template .collegelist}
+  <div class="mdl-card__supporting-text">
```

Comment:
I think here I will make a table to represent the college. Maybe having a generic list view template and then extending that for both colleges and scholarships might be a good idea. Here is the table in 
[MDL](https://getmdl.io/components/#tables-section). If we want pagination this [boostrap component](https://getbootstrap.com/docs/4.5/components/pagination/) could work or we could make our own custom component, doesn't really matter either way. This is more prevalent to the MVP milestone of course.

Reply:
If we use the MDL table, I wonder how we would bind event handlers to the individual cells in the table?

Reply:
Yeah we'll call a generic list view here in both template. I think we'll need a new design doc for this.